giticket/giticket.py

```diff
@@ -71,6 +71,7 @@ def main(argv=None):
     args = parser.parse_args(argv)
     regex = args.regex or r'[A-Z]+-\d+'  # noqa
     format_string = args.format or '{ticket} {commit_msg}' # noqa
+    format_string = format_string.replace('\\n', '\n')
```

Comment:
Thanks.

It would be good to have [tests](https://github.com/milin/giticket/blob/master/tests/test_giticket.py) for this case as well.